aspnetcore/includes/mvc-intro/search3.md

```diff
@@ -55,4 +55,10 @@ Update `Index.cshtml` as follows:
 
 [!code-HTML[Main](../../tutorials/first-mvc-app/start-mvc/sample/MvcMovie/Views/Movies/IndexFormGenreNoRating.cshtml?highlight=1,15,16,17,28,31,34,37,43)]
 
-Test the app by searching by genre, by movie title, and by both.
\ No newline at end of file
+Examine the lambda expression used in the following `HTML Helper`:
+
+ `@Html.DisplayNameFor(model => model.movies[0].Title)`
+ 
+In the preceding code, the HTML Helper `DisplayNameFor` inspects the lambda expression `model => model.movies[0].Title` to determine the display name. The lambda expression is not evaluated, that's why you don't get an access violation when `model`, `model.movies`, or `model.movies[0]` are `null` or empty. When the lambda expression is evaluated, it will be run against the model.
```

Comment:
Other than adding "method" after `` `DisplayNameFor` ``, looks good

Completely up to you but you might want to rephrase "When the lambda expression is evaluated" so it's clear you're talking about another HTML helper method and the view's `Model`.